cluster/log-dump/log-dump.sh

```diff
@@ -419,6 +419,10 @@ function dump_nodes() {
   all_selected_nodes+=( "${windows_node_names[@]}" )
 
   proc=${max_dump_processes}
+  start="$(date +%s)"
+  # This is just a rough estimation of the timeout length which may be
```

Comment:
I'd start with a  sentence explaining what log_sump_ssh_timeout is supposed to do and then explain that it may not work accurately. So something like:

```
log_dump_ssh_timeout is the max number of seconds the log dumping over ssh operation can take. 
Please note that logic enforcing the timeout is only a best effort. The actual time of the operation may be longer due to ....
```

```

Reply:
Done.